### PR TITLE
Home task2

### PR DESCRIPTION
**Pull request by SudharshanThammaiahgari**
These are coding questions that are part of Home Task 2. Could you please review it ?

### CODE REVIEW

File: commaDelimited.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            delimitedString += arr[i] + ",";
+        } else {
+            delimitedString += arr[i];
+        }
+    }
+    return delimitedString;
+}
+
+var arr = ["h", "o", "l", "a"]
+var res = commaDelimited(arr);
```
Comment (nrsedat) [inline commaDelimited.js]: Array.prototype.join.call(arr)

File: mathOperations.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            res = num1 * num2;
+            break;
+        case '/':
+            res = num1 / num2;
+            break;
+        default:
+            console.log("Specify a valid operator");
+            return;
+    }
+    console.log("(\'" + operator + "\', " + num1 + ", " + num2 + ") " + "--> " + res);
```
Comment (nrsedat) [inline mathOperations.js]: suggestion: Move the console statement outside of the function. Return the result from the function

File: commaDelimited.js (hunk 1 above)
Comment (SudharshanThammaiahgari) [reply commaDelimited.js]: Done Ratheesh. But I thought to implement without any pre-built function.

File: mathOperations.js (hunk 2 above)
Comment (SudharshanThammaiahgari) [reply mathOperations.js]: Sure Ratheesh.

**Review by nrsedat — APPROVED** [review]

**Pull request merged**